packages/clerk-js/src/core/__tests__/clerk.test.ts

```diff
@@ -486,6 +487,15 @@ describe('Clerk singleton', () => {
         lastActiveToken: { getRawString: () => 'mocked-token' },
         tasks: [{ key: 'org' }],
         currentTask: { key: 'org', __internal_getUrl: () => 'https://foocorp.com/add-organization' },
+        reload: jest.fn(() =>
+          Promise.resolve({
+            id: '1',
+            status: 'pending',
+            user: {},
+            tasks: [{ key: 'org' }],
+            currentTask: { key: 'org', __internal_getUrl: () => 'https://foocorp.com/add-organization' },
```

Comment:
For posterity: I'm always just a little nervous of including random routable URLs in tests. 

Truly unlikely security threat potential: If we actually GET this URL, and someone knows we're getting it, they could register the domain and have it serve some exploit that gets us to leak our CI secrets.

I'm not even proposing you change this - it's the pattern laid down elsewhere in the test - but when in doubt, I like `example.com` or cnames of it. It's not routable. `https://foocorp.example.com/add-organization` is safer in my book.

Reply:
Makes sense! Also just to note that our tests only run if the triggering actor (owner of the PR) has the GitHub org permissions to access secrets, otherwise it'll fail 

For URLs like those that are used to trigger navigation rather than HTTP requests, then I think it's fine!